### PR TITLE
Shladkyi - Add "Buy Product" test and RozetkaBasketPage

### PR DESCRIPTION
**Pull request by SerhiiHladkyi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,7 @@ buildNumber.properties
 .mvn/wrapper/maven-wrapper.jar
 .idea/
 *.iml
+/build/
 
+
+/build/
```
Comment (VladimirPetrat) [inline .gitignore]: Two times

File: src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        chosenProduct.returnToCategoryPage(1);
+        Assert.assertTrue(catalogCategoryItem.getPageTitle().contains("Комп'ютери"));
+    }
+
+    @Test
+    public void testAddingProductToBasket() {
+        RozetkaProductPage productPage = rozetkaHomePage
+                .navigateToCategoryPage(6)
+                .navigateToProductCategoryPage(1)
+                .navigateToProductByNumber(0);
```
Comment (VladimirPetrat) [inline src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java]: Can replace with navigateToFirstProduct()

Comment (SerhiiHladkyi) [reply src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java]: This belongs to my test case.

Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java]: use specific test data

File: src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        RozetkaBasketPage basketPage = productPage.buyProduct();
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java]: this just adds product to the basket, right?

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request closed without merging**